es-CL/firefox/enterprise.ftl

```diff
@@ -13,7 +13,7 @@ firefox-enterprise-downloads = Descargas
 firefox-enterprise-get-firefox-for-your-enterprise = Obtén { -brand-name-firefox } para tu empresa
 # Variables:
 #   $promise (url) - link to https://support.mozilla.org/kb/choosing-firefox-update-channel
```

Comment:
Nit: we should probably correct the variable name to `$url` in this comment

Reply:
@craigcook  updated in [9199988](https://github.com/mozilla-l10n/www-l10n/pull/271/commits/9199988e77eb11c33a3d60a43908a42b3bcb7945) (what a commit hash!)